scripts-available/CDB_Conf.sql

```diff
@@ -0,0 +1,58 @@
+-- This will trigger NOTICE if CDB_CONF already exists
+DO LANGUAGE 'plpgsql' $$
+BEGIN
+    CREATE TABLE IF NOT EXISTS cartodb.CDB_CONF ( PARAM TEXT PRIMARY KEY, CONF TEXT NOT NULL );
+    EXECUTE format('GRANT SELECT ON cartodb.CDB_CONF TO %s', cartodb.CDB_Organization_Member_Group_Role_Member_Name());
+END
+$$;
+
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Conf_SetConf(param text, conf text)
```

Comment:
Previous line 5 grants select to all organization users. Only postgres is allowed to write.